bin/export-import-data

```diff
@@ -130,11 +134,11 @@ sub import {
 
     foreach (@{$out->{templates}}) {
         my $existing = $body->response_templates->search({ title => $_->{title} })->single;
-        if ($existing) {
+        if ($existing && !$update_templates) {
             warn "Template with title $_->{title} already exists, skipping";
             next;
         }
-        my $template = $body->response_templates->new({
+        my $template = $body->response_templates->find_or_new({
```

Comment:
This won't update the text or status of any existing templates unfortunately. `find_or_new` will (I believe) only search on unique columns, meaning it'll match on `body_id` and `title` then skip over the values for `text` and `state`.

Reply:
sigh, yes :(